pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/SingularFixRaceQuestionTypo.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.jdbi.v3.core.Handle;
+
+import java.io.File;
+import java.nio.file.Path;
+
+@Slf4j
+public class SingularFixRaceQuestionTypo implements CustomTask {
+    private static final String STUDY_GUID = "singular";
+    private static final String DATA_FILE = "patches/singular-race-question-prompt-fix.conf";
+    private static final String TEMPLATE_VARIABLE_NAME = "what_is_your_race_prompt_p1";
+    private static final String QUESTION_STABLE_ID = "RACE";
+    private static final String LANGUAGE_CODE_EN = "en";
+    private static final String QUESTION_TEMPLATE_VARIABLE_SUBSTITUTION_VALUE = "UPDATE " +
+            "  template_variable tv " +
+            "  join question q on q.question_prompt_template_id = tv.template_id natural " +
+            "  join question_stable_code qsc natural " +
+            "  join language_code lc " +
+            "  join study_activity sa on sa.study_activity_id = q.study_activity_id " +
+            "  join i18n_template_substitution its on its.template_variable_id = tv.template_variable_id " +
+            "  join umbrella_study us on us.umbrella_study_id = sa.study_id " +
+            "set " +
+            "  substitution_value = ?" +
+            "where " +
+            "  tv.variable_name = ? " +
+            "  AND qsc.stable_id = ? " +
+            "  AND lc.iso_language_code = ? " +
+            "  AND us.guid = ?;";
```

Comment:
The `+` operator needs to be on the new line, not on the end of the preceding one (this is a checkstyle error)

Reply:
Oh geez! The IntelliJ was nice enough to convert my pasted string into multi-line concatenations for me.
I hope I don't mess it up trying to manually make checkstyle happy!

Reply:
ffe2d5b76